blog/2023-10-1-cloudgaming/index.md

```diff
@@ -0,0 +1,83 @@
+---
+slug: Cloud-gaming
+title: Exploring the Future of Gaming
+author: [Om Hinge]
```

Comment:
This line is breaking the deployment.

That's not how you add authors. Check the posting guidelines blog for details.

Reply:
ohhk